src/features/surveys/components/surveyForm/SurveyPrivacyPolicy.tsx

```diff
@@ -22,6 +22,9 @@ export type SurveyPrivacyPolicyProps = {
 
 const SurveyPrivacyPolicy: FC<SurveyPrivacyPolicyProps> = ({ survey }) => {
   const messages = useMessages(messageIds);
+  const privacyUrl =
+    process.env.ZETKIN_PRIVACY_POLICY_LINK || messages.surveyForm.policy.link();
```

Comment:
This works, because this is a React Server Component, i.e. it's rendered on the server where environment variables are available. If this component is ever rendered by the client, it will not work.

In those cases, we need to use `useEnv()` instead. But that wouldn't work in a RSC (like this one) because it relies on context.

So this is the right way to do it, but also error-prone because it's easy to forget to change to `useEnv()` if this component is ever modified to be client-rendered. I'm mentioning it here for future forensics into potential bugs.